electron-app/package.json

```diff
@@ -16,20 +35,99 @@
+    "vscode-builtin-scss": "https://open-vsx.org/api/vscode/scss/1.39.1/file/vscode.scss-1.39.1.vsix",
+    "vscode-builtin-shaderlab": "https://open-vsx.org/api/vscode/shaderlab/1.39.1/file/vscode.shaderlab-1.39.1.vsix",
+    "vscode-builtin-shellscript": "https://open-vsx.org/api/vscode/shellscript/1.39.1/file/vscode.shellscript-1.39.1.vsix",
+    "vscode-builtin-sql": "https://open-vsx.org/api/vscode/sql/1.39.1/file/vscode.sql-1.39.1.vsix",
+    "vscode-builtin-swift": "https://open-vsx.org/api/vscode/swift/1.39.1/file/vscode.swift-1.39.1.vsix",
+    "vscode-builtin-theme-abyss": "https://open-vsx.org/api/vscode/theme-abyss/1.39.1/file/vscode.theme-abyss-1.39.1.vsix",
+    "vscode-builtin-theme-defaults": "https://open-vsx.org/api/vscode/theme-defaults/1.39.1/file/vscode.theme-defaults-1.39.1.vsix",
+    "vscode-builtin-theme-kimbie-dark": "https://open-vsx.org/api/vscode/theme-kimbie-dark/1.39.1/file/vscode.theme-kimbie-dark-1.39.1.vsix",
+    "vscode-builtin-theme-monokai": "https://open-vsx.org/api/vscode/theme-monokai/1.39.1/file/vscode.theme-monokai-1.39.1.vsix",
+    "vscode-builtin-theme-monokai-dimmed": "https://open-vsx.org/api/vscode/theme-monokai-dimmed/1.39.1/file/vscode.theme-monokai-dimmed-1.39.1.vsix",
+    "vscode-builtin-theme-quietlight": "https://open-vsx.org/api/vscode/theme-quietlight/1.39.1/file/vscode.theme-quietlight-1.39.1.vsix",
+    "vscode-builtin-theme-red": "https://open-vsx.org/api/vscode/theme-red/1.39.1/file/vscode.theme-red-1.39.1.vsix",
+    "vscode-builtin-theme-solarized-dark": "https://open-vsx.org/api/vscode/theme-solarized-dark/1.39.1/file/vscode.theme-solarized-dark-1.39.1.vsix",
+    "vscode-builtin-theme-tomorrow-night-blue": "https://open-vsx.org/api/vscode/theme-tomorrow-night-blue/1.39.1/file/vscode.theme-tomorrow-night-blue-1.39.1.vsix",
+    "vscode-builtin-typescript": "https://open-vsx.org/api/vscode/typescript/1.39.1/file/vscode.typescript-1.39.1.vsix",
+    "vscode-builtin-typescript-language-features": "https://open-vsx.org/api/vscode/typescript-language-features/1.39.1/file/vscode.typescript-language-features-1.39.1.vsix",
+    "vscode-builtin-vb": "https://open-vsx.org/api/vscode/vb/1.39.1/file/vscode.vb-1.39.1.vsix",
+    "vscode-builtin-icon-theme-seti": "https://open-vsx.org/api/vscode/vscode-theme-seti/1.39.1/file/vscode.vscode-theme-seti-1.39.1.vsix",
+    "vscode-builtin-xml": "https://open-vsx.org/api/vscode/xml/1.39.1/file/vscode.xml-1.39.1.vsix",
+    "vscode-builtin-yaml": "https://open-vsx.org/api/vscode/yaml/1.39.1/file/vscode.yaml-1.39.1.vsix",
+    "vscode-editorconfig": "https://open-vsx.org/api/EditorConfig/EditorConfig/0.14.4/file/EditorConfig.EditorConfig-0.14.4.vsix",
+    "vscode-clangd": "https://open-vsx.org/api/llvm-vs-code-extensions/vscode-clangd/0.0.21/file/llvm-vs-code-extensions.vscode-clangd-0.0.21.vsix",
+    "cdt-gdb-vscode": "https://open-vsx.org/api/eclipse-cdt/cdt-gdb-vscode/0.0.91/file/eclipse-cdt.cdt-gdb-vscode-0.0.91.vsix"
```

Comment:
Just to be clear, is this adding all the language plugins to the example apps? go, csharp, ruby, rust, vb(!), etc?

Reply:
For the most part just the TextMate grammar is provided through these "built-in" VS Code extensions. It means they're tiny and are useful to at least get coloring in the editor for the main languages and file types.

There are a few example where full language support is included, like for JS/TS - these tend to be named "<something>-language-features.vsix", and complete the base extension providing the grammar.

Reply:
In the not-so-distant future, I plan to have "extension pack(s)" where we can pull all or most of the "built-ins" from a single entry vs ~50+ as it's the case now. It will be cleaner while giving us similar baseline feature as VS Code.